lib/membrane_webrtc_plugin/sdp.ex

```diff
@@ -287,6 +306,12 @@ defmodule Membrane.WebRTC.SDP do
     ssrc = Media.get_attribute(sdp_media, :ssrc)
     ssrc = if ssrc == nil, do: nil, else: ssrc.id
 
+    rids =
+      Media.get_attributes(sdp_media, "rid")
+      |> Enum.map(fn {_attr, rid} ->
+        rid |> String.split(" ", parts: 2) |> hd
+      end)
```

Comment:
We should use parentheses event if function takes 0 arguments or 1 in case of pipe [(guide)](https://github.com/christopheradams/elixir_style_guide/blob/master/README.md#function-calls-and-parentheses)
```suggestion
    rids =
      sdp_media
      |> Media.get_attributes("rid")
      |> Enum.map(fn {_attr, rid} ->
        rid |> String.split(" ", parts: 2) |> hd()
      end)
```